deploy.bat

```diff
@@ -1,21 +0,0 @@
-@echo off
```

Comment:
Can this be updated rather than removed? or does a deploy now work well on a windows dev container?